CHANGELOG.md

```diff
@@ -270,6 +270,7 @@ to detail this new feature and how state transitions occur.
 * (keys) Fix ledger custom coin type support bug.
 * (x/gov) [\#5107](https://github.com/cosmos/cosmos-sdk/pull/5107) Sum validator operator's all voting power when tally votes
 * (rest) [\#5212](https://github.com/cosmos/cosmos-sdk/issues/5212) Fix pagination in the `/gov/proposals` handler.
+* (x/bank) [\#5531](https://github.com/cosmos/cosmos-sdk/issues/5531) Added missing amount event to MsgMultiSend, emitted for each output.
```

Comment:
Sorry to be a hassle here, but we've recently released 0.38, so this now belongs under the new `Unreleased` section.

Reply:
i.e.

```
## [Unreleased]

### Bug Fixes

* (x/bank) [\#5531](https://github.com/cosmos/cosmos-sdk/issues/5531) Added missing amount event to MsgMultiSend, emitted for each output.
```

Reply:
Oh ok, I was confused because that's what the PR checklist asked me to do :) I understand now.

Reply:
Oh ok no problem :)